agent_horizon_s: 1.75
radius_m: 15.0
num_semantic_classes: 20
min_distance_m: 0.5
max_distance_m: 2.5
distance_bins: 30
min_correspondences: 7
min_inliers: 7
agent:
    min_time_separation_s: 150.0
    min_score: 0.059
    type: "L1"
object:
    min_time_separation_s: 150.0
    min_score: 0.66
    min_registration_score: 0.9
    max_registration_matches: 5
    min_score_ratio: 0.5
    min_match_separation_m: 5.0
    type: "L1"
places:
    min_time_separation_s: 150.0
    min_score: 0.75
    type: "L1"
teaser:
    noise_bound: 0.1
    cbar2: 1.0
    rotation_gnc_factor: 1.4
    rotation_max_iterations: 100
    rotation_cost_threshold: 1.0e-6
    kcore_heuristic_threshold: 0.5
    inlier_selection_mode: 0 # (pcm_exact = 0, pcm_heu = 1, kcore_heu = 2, none = 3)
    max_clique_time_limit: 10